plugin

```diff
@@ -226,6 +226,7 @@
   "nielsfaber/alarmo-card",
   "nielsfaber/scheduler-card",
   "Nyaran/myjdownloader-card",
+  "luixal/lovelace-media-source-image-card",
```

Comment:
This is too far of for me to do in a suggestion, you need to sort this alphabetically.

Reply:
Wow. I though I had sorted it. Sorry, will do.